probe-rs/src/architecture/riscv/mod.rs

```diff
@@ -159,9 +159,11 @@ impl<'probe> Riscv32<'probe> {
                     .read_core_reg(core.registers().get_argument_register(1).unwrap().id())?
                     .try_into()?;
 
-                if let Some(command) = decode_semihosting_syscall(a0, a1) {
-                    reason = HaltReason::Breakpoint(BreakpointCause::Semihosting(command));
-                }
+                tracing::info!("Semihosting found pc={pc:#x} a0={a0:#x} a1={a1:#x}");
```

Comment:
Why did you decide to make this an `info`? The preceding step uses a `debug`, and to me it feels as if the two messages belong on the same level of logging.

Reply:
I actually copied this from https://github.com/probe-rs/probe-rs/blob/master/probe-rs/src/architecture/arm/core/cortex_m.rs#L190C6-L219 .
Semihosting check is executed quite often (on each status() invocation). But "semihosting found" occurs only when there's an actual semihosting call, so I think it can make sense that we have two different levels here.

Reply:
I would like to keep this line as it, for the reason outlined above (occurrence frequency).